lms/static/js/verify_student/views/webcam_photo_view.js

```diff
@@ -241,12 +245,18 @@
             );
             $( this.el ).html( renderedHtml );
 
+            var resetBtn = this.$el.find('#webcam_reset_button');
+            var captureBtn = this.$el.find('#webcam_capture_button');
```

Comment:
Nitpick 1: all vars should be called at the top of the function. As per line 5 of the file you only need to declare var once and then comma separate the variables.

Nitpick 2: It is a js convention to name DOM element variables starting with $, so these should be $resetBtn and $captureBtn